src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="personal-information-fieldset">
+        <legend>Personal information</legend>
+        <label for="surname" class="surname-label">Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            class="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label for="name" class="name-label">Name:
```

Comment:
Let's fix this formatting issue throughout your code

```suggestion
        <label for="name" class="name-label">
          Name:
```